packer/builder/azure/arm/step_power_off_compute_test.go

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Open Technologies, Inc.
+// All Rights Reserved.
+// Licensed under the Apache License, Version 2.0.
+// See License.txt in the project root for license information.
+
+package arm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/Azure/packer-azure/packer/builder/azure/common/constants"
+	"github.com/mitchellh/multistep"
+)
+
+func TestStepPowerOffComputeShouldFailIfValidateFails(t *testing.T) {
```

Comment:
You're probably going to have quite a few of these because I copy and pasted the test cases as a starting point. I cleaned up prefix, but not the suffix.